plugins/inputs/systemd_units/README.md

```diff
@@ -29,22 +57,30 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 [[inputs.systemd_units]]
   ## Set timeout for systemctl execution
   # timeout = "1s"
-  #
+
+  ## Select the systemctl subcommand to use to gather information.
+  ## Using `list-units` is the option with the broadest compatibility.
+  ## Using `show` will get more information but may fail to list all units on
+  ## some systems.
+  # subcommand = "list-units"
+
   ## Filter for a specific unit type, default is "service", other possible
   ## values are "socket", "target", "device", "mount", "automount", "swap",
   ## "timer", "path", "slice" and "scope ":
   # unittype = "service"
-  #
+
   ## Filter for a specific pattern, default is "" (i.e. all), other possible
   ## values are valid pattern for systemctl, e.g. "a*" for all units with
   ## names starting with "a"
   # pattern = ""
-  ## pattern = "telegraf* influxdb*"
-  ## pattern = "a*"
+  # pattern = "telegraf* influxdb*"
+  # pattern = "a*"
```

Comment:
In general the config options should show the default value. We like to avoid adding examples below, and rather keep them in the comments. Let's remove these and either keep the current comments or add additional examples.